demos/accordion/default.html

```diff
@@ -4,15 +4,11 @@
 	<meta charset="utf-8">
 	<title>jQuery UI Accordion - Default functionality</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/accordion.js"></script>
 	<link rel="stylesheet" href="../demos.css">
-	<script>
-	$(function() {
+	<script src="../require-config.js"></script>
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js">
```

Comment:
Let's merge `require-config.js` and `bootstrap.js`, use `require.config()` in it, after loading `require.js`.